### PR TITLE
1.5/1750 market package updates vvv e

### PR DESCRIPTION
**Pull request by neonleann**
https://trello.com/c/Wi0yFngm/1750-market-package-updates-vvv-e

All the changes I was able to make to Market. Details in comment on card.

I also ran npm audit fix, which bought vulnerabilities down to:
found 23 vulnerabilities (2 low, 11 moderate, 10 high)
from
found 90 vulnerabilities (2 low, 48 moderate, 40 high) - before I started doing anything.

There are a few changes to tests to make them work, as well. All 56 pass right now.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ module.exports = {
     env: {
         browser: true,
     },
-    extends: ["plugin:vue/essential", "prettier", "prettier/standard"],
+    extends: ["plugin:vue/essential", "prettier"],
```
Comment (charlesstrange2) [inline .eslintrc.js:13]: did prettier/standard go away?

File: package.json [hunk 2]
```diff
@@ -1,44 +1,44 @@
...
         "copy-webpack-plugin": "^6.3",
         "cross-env": "^7.0.3",
         "css-loader": "^5.0",
-        "eslint": "^7.15.0",
-        "eslint-config-prettier": "^7.0.0",
-        "eslint-plugin-prettier": "^3.2.0",
+        "eslint": "^8.9.0",
+        "eslint-config-prettier": "^8.3.0",
+        "eslint-plugin-prettier": "^4.0.0",
         "eslint-plugin-standard": "^5.0.0",
```
Comment (charlesstrange2) [inline package.json:27]: Do we need standard?

File: package.json [hunk 3]
```diff
@@ -52,7 +52,7 @@
     "scripts": {
         "dev": "cross-env NODE_ENV=development webpack-dev-server --open --hot",
         "build": "cross-env NODE_ENV=production webpack --mode production --progress --hide-modules --display-error-details",
-        "test": "testcafe chrome,firefox ./tests/*/*.test.js --app 'cross-env NODE_ENV=development webpack-dev-server' --app-init-delay 10000",
+        "test": "testcafe chrome ./tests/*/*.test.js --app 'cross-env NODE_ENV=development webpack-dev-server' --app-init-delay 10000",
```
Comment (charlesstrange2) [inline package.json:55]: there is an argument that we should test on safari, as mobile OS will use that engine BUT, that means having safari on each machine. so let's not.

**Review by charlesstrange2 — APPROVED** [review]
Okedoke, give it a spin :whale:

File: .eslintrc.js (hunk 1 above)
Comment (neonleann) [reply .eslintrc.js:13]: It did - https://github.com/prettier/eslint-config-prettier/blob/main/CHANGELOG.md.

**Pull request merged**